site/en/docs/extensions/mv3/xhr/index.md

```diff
@@ -214,6 +182,13 @@ If you modify the default [Content Security Policy][12] for your extension by ad
 you'd like to connect are allowed. While the default policy doesn't restrict connections to hosts,
 be careful when explicitly adding either the `connect-src` or `default-src` directives.
 
+### What about XMLHttpRequest?
+
+The `XMLHttpRequest()` (XHR) API is supported for backwards compatability, but new work should favor the use of `fetch()` wherever possible. The `fetch()` API allows you to make network requests similar to XMLHttpRequest, but has the additional benefit of performing asynchronously via promises. 
```

Comment:
Was there a specific reason for adding this? I think we probably don't need it - `XMLHttpRequest` is a web API, not an extension API, and so I think it's expected understanding and more confusing to call it out.

Reply:
Yes. `fetch()` was created specifically for service workers and follows a broader web trend away from synchronous operations. Manifest V3 doesn't allow remotely hosted code, but it does allow [calling services](https://developer.chrome.com/docs/extensions/migrating/improve-security/#remove-remote-code:~:text=Externalized%20logic%20with%20a%20remote%20service). For that, developers may want and probably should use the extension service worker as a network proxy. That was the reason service workers were created in the first place. In light of this, does it make sense code outside the service worker to use a different interface?

With apologies to Ian (this is largely based on my suggestion) I think I should have suggested something slightly different for this. It should cover these points without making it sound as though the web platform is about to deprecate XHR. It definitely is not.

* Prefer `fetch()` because it's asynchronous and was made explicitly for service workers.
* `fetch()` handlers in a service worker will intercept `XMLHttpRequest()` calls.